finality-aleph/src/session_map.rs

```diff
@@ -14,19 +14,17 @@ use tokio::sync::{
     RwLock,
 };
 
-use crate::{
-    first_block_of_session, session_id_from_block_num, ClientForAleph, SessionId, SessionPeriod,
-};
+use crate::{session::SessionInfo, ClientForAleph, SessionId, SessionPeriod};
 
 const PRUNING_THRESHOLD: u32 = 10;
 type SessionMap = HashMap<SessionId, SessionAuthorityData>;
 type SessionSubscribers = HashMap<SessionId, Vec<OneShotSender<SessionAuthorityData>>>;
 
-pub trait AuthorityProvider<B> {
+pub trait AuthorityProvider<N> {
```

Comment:
Eh? What happened here?

Reply:
I changed it to `N`, as this was not generic over `Block`, but over number. Not related to the PR, but wanted to squeeze that in somewhere.

Reply:
Ah, could you then also change the `block` in the calls to `block_number` or something, to make this clearer?